tests/ci/docker_images/linux-aarch/ubuntu-20.04_clang-10x/Dockerfile

```diff
@@ -1,28 +1,15 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-FROM arm64v8/ubuntu:20.04
+FROM ubuntu-20.04-aarch:base
 
 SHELL ["/bin/bash", "-c"]
 
-ENV DEBIAN_FRONTEND=noninteractive
-
 RUN set -ex && \
     apt-get update && \
     apt-get -y --no-install-recommends upgrade && \
```

Comment:
Tiny NP: in all of these I think we can save time and skip calling upgrade since we do that in the base. I'm not sure if we need the update as well to get the latest package information, we might need to since we delete `/var/lib/apt/lists` and that seems relevant.

Reply:
Yes, I kept it because I am not sure the side effects.